src/styles/blocks/container.scss

```
@@ -0,0 +1,17 @@
+.container {
+  display: grid;
+  grid-template-columns: repeat(auto-fill, minmax($card-heigth, 1fr));
```

using a card-height variable for width is a little bit confusing

```suggestion
  grid-template-columns: repeat(auto-fill, minmax($card-size, 1fr));
```